wire-ios-request-strategy/Tests/TestPlans/AllTests.xctestplan

```diff
@@ -0,0 +1,75 @@
+{
+  "configurations" : [
+    {
+      "id" : "1F8A3CDC-E12B-4921-9BD0-0ADB68899DC0",
+      "name" : "Configuration 1",
+      "options" : {
+
+      }
+    }
+  ],
+  "defaultOptions" : {
+    "commandLineArgumentEntries" : [
+      {
+        "argument" : "-com.apple.CoreData.ConcurrencyDebug 1"
+      }
+    ],
+    "environmentVariableEntries" : [
+      {
+        "enabled" : false,
+        "key" : "LIBDISPATCH_COOPERATIVE_POOL_STRICT",
+        "value" : "1"
+      }
+    ],
+    "language" : "en",
+    "region" : "DE",
+    "testExecutionOrdering" : "random"
+  },
+  "testTargets" : [
+    {
+      "skippedTests" : [
```

Comment:
Setting these here, removing it from the scheme `wire-ios-request-strategy/WireRequestStrategy.xcodeproj/xcshareddata/xcschemes/WireRequestStrategy.xcscheme`